### ISSUE TITLE
dev: metadata of to do items

### ISSUE THREAD

**Issue by LauraErhard** [locdb/locdb-frend#276]
The metadata for the different to do items seems to be largely missing. 

The journal issue has been automatically created, but there is no data available there:
![metadata_journalissue](https://user-images.githubusercontent.com/32642653/39304541-46c8f374-495a-11e8-8cb9-c5c521dfe064.png)
We would at least expect to see the journal title, issue number and the right publication year (right now it is the uploading date).

The journal article seems to be always missing the contributors and the DOI URL doesn't have an identifier typ anymore (I think we called that URL before, but it doesn't exist now)
![metadata_journalarticle](https://user-images.githubusercontent.com/32642653/39304705-d9eff0ee-495a-11e8-94f6-053a678f1b7f.png)

The edited_book seems to be missing the contributors as well. (By the way, don't we call edited_book just book now?)
![metadata_editedbook](https://user-images.githubusercontent.com/32642653/39304774-1f3dbdc0-495b-11e8-8747-81aa20f79121.png)

The book chapters are the same as the journal issues. We would like to see the chapter title, author and number if they are available. But at least the right date should be inherited.

The monograph seem to be missing the contributors too.

**Comment by lgalke** [comment]
The contributors seem to not make it correctly into the intial form. We well check this. Regarding the other meta-data: for now you can only edit the *own* meta-data in the form and not the meta-data of the parent.

We might reconsider this at some point, but there is one important thing to keep in mind:
When the properties of a container are edited within one of its "child" resource, there is no way that these changes will be reflected in the container itself or in other children of the same container. Thus, it would lead to data inconsistencies. I assume that editing the container meta data for every single resource inside the container (multiple times in the same way) is both not feasible and not desirable.
Please let me know if I'm wrong.

Of course, on the other hand, just displaying the container metadata (non-editable) is a valid option, we will tackle this in the same manner as in #275 .

**Comment by LauraErhard** [comment]
In every contributors field is now an `UNK` filled in. Is this part of the process?
![contributors](https://user-images.githubusercontent.com/32642653/39361416-a6eb0242-4a22-11e8-952d-916abc8af335.png)

**Comment by lgalke** [comment]
This is part of the proper treatment of extracting the name for display from the different name properties (firstname, lastname, nameString). We now display "UNK" (for [UNKNOWN](https://www.abbreviations.com/abbreviation/UNKNOWN)) if none of these attributes is given.. which means that in the data there is basically only some identifier. We still need to find a good way to properly deal with those identifiers I guess. So for now "UNK" is the fallback value if there is basically no name, before it was just empty "".

**Comment by LauraErhard** [comment]
Okay, thanks for clearing this up.
But you are still checking why the contributors didn't make it into the initial form, right?

**Comment by anlausch** [comment]
I checked the raw data of the journal issue and there is some more information we can display available, e.g.: "```
journal_title": "Acta Sociologica",
  "journalVolume_number": "54",
```
Let me know if you encounter missing meta data in the raw versions.

**Comment by lgalke** [comment]
Showing these properties depends on #275 @anlausch, but what about the name. To me it looks strange that there are apparrently the roles of the contributors defined (see Lauras screenshot) but the names got lost. I double-checked our code, but we do properly extract the Name, i.e. when anything of `givenName`, `firstName` or `nameString` is present, we display it (and also copy it to the form).

**Comment by anlausch** [comment]
I will also check this in detail.

**Comment by anlausch** [comment]
There was a problem with saving the data. It should be resolved now (both for meta data of the references and the authors).

**Comment by lgalke** [comment]
This should be resolved via #290
